packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -716,8 +826,25 @@ export const BaseInput = React.forwardRef<HTMLInputElement, BaseInputProps>(
             validationState={validationState}
             currentInteraction={currentInteraction}
             isLabelLeftPositioned={isLabelLeftPositioned}
+            showAllTags={showAllTags}
+            setShowAllTagsWithAnimation={setShowAllTagsWithAnimation}
+            ref={inputWrapperRef as any}
           >
             <BaseInputVisuals leadingIcon={leadingIcon} prefix={prefix} isDisabled={isDisabled} />
+            <BaseInputTagSlot
+              tags={tags}
+              showAllTags={showAllTagsWithAnimation}
+              setFocusOnInput={() => {
+                if (ref && 'current' in ref) {
```

Comment:
nit: Make it easier?
```suggestion
                if (ref && ref?.current) {
```

Reply:
<img width="578" alt="image" src="https://github.com/razorpay/blade/assets/30949385/613f8b42-6b23-4482-8306-d825d895684e">

Its the does not exist error so ref?.current doesn't make TS happy 😢